### ISSUE TITLE
Inconsistent dependencies of Cookie and openssl

### ISSUE THREAD

**Issue by matsbror** [rustless/rustless#80]
I just discovered rustless and wanted to give it a try on Linux Ubuntu 16.04. I have rustc 1.13 and cargo 0.13.0-nightly.

The build of rustless fails because of conflicting dependences on two different versions of openssl. As far as I can tell from the Cargo.lock file, these come from dependencies on two different versions of cookie. rustless wants cookie 0.4.0 and hyper wants cookie 0.2.5.

Since I am new to the rust world, I have no clue as to how to resolve this.

**Comment by mmacedoeu** [comment]
This crate doesn't look active. I have fixed this clash on my forked and submited a pull request. You can use my version using this config on your cargo.toml:

```
[dependencies.rustless]
git = "https://github.com/mmacedoeu/rustless"
```

**Comment by s-panferov** [comment]
Fixed in v0.10.0
